examples/make_assets/structure_viz.py

```diff
@@ -31,11 +30,20 @@
 
 
 # %% Plot some disordered structures in 2D
+mp_api_key = os.environ.get("MP_API_KEY")
+if mp_api_key is None:
+    raise RuntimeError("api key is None")
+
+elif mp_api_key is not None and len(mp_api_key.strip()) == 0:
+    raise RuntimeError("mp api key is empty")
```

Comment:
@janosh  can you have another look at the API_KEY secret (I don't have access to that)? [It looks like being empty](https://github.com/janosh/pymatviz/actions/runs/11071283116/job/30762788796?pr=207)?
> RuntimeError: mp api key is empty

Reply:
i can't see it either. once you set a secret you can only change it's value, not see the current value. i just reset the secret to the what i think it was before so let's see if that makes a difference

Reply:
Still no luck. I'm not seeing the `test.yml` or the template from your `workflows` being changed at all recently, so super confused what is causing this failure.

 I would have another look tomorrow because I don't have my laptop with me ATM (which I usually use to code), so everything feels very unnatural :)

Another thread I haven't got to look at, not sure if it's related at all: [ Github Actions reusable workflow doesn't recognize secrets #69082 ](https://github.com/orgs/community/discussions/69082)

Reply:
> not sure if it's related at all

don't think so, the `test-scripts` job doesn't have a reusable workflow

Reply:
I believe it's related to [secrets access from forked repos](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions?tool=webui):
> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.

As you have noticed both this PR and #210 were created from our forks.

However above documentation doesn't provide any workaround for this :(

Reply:
that sounds like it could be the culprit. probably the thing to do is to rather than fetch those structures on the fly, save them as JSON to `tests/files/structs` and load them from there in this script (while leaving the code that generated those structures files in place but behind `if no os.path.isfile(...)` as documentation)

Reply:
You always have helpful suggestions :) How is 95614f26fd63f0213b8cd856bff9cae6b06679b1 looking? I don't really like to manually download and save files or such, it's always better to automate this